src/code/code_800C3C20.c

```diff
@@ -5,9 +5,9 @@ u8 sSfxBankIds[] = {
 };
 
 void AudioMgr_StopAllSfx(void) {
-    u32 i;
+    volatile u8* bankIdPtr;
```

Comment:
it was needed for mm which suggests it will be needed for our retail as well

Reply:
Apparently having `bankIdPtr` as `volatile` or not has no effect on this implementation of this function (i.e. OoT's implementation, which is different from MM's, MM has an inner `if` https://decomp.me/scratch/fxTS7 )

Since this function is probably no different in other OoT versions, (i.e. previous versions since OoT MQ debug is one of the most "recents"), I think it won't ever need `volatile` and we can remove it at least for now (instead of carrying it forever :p)